dev/update_qs.R

```diff
@@ -0,0 +1,107 @@
+#' Program Name : update_qs.R
+
+#' Description : create VFQ questionnaire test data for ADMIRAL Ophtha
+
+#' Input : raw_qs
+
+#' Author : Ritika Aggarwal
```

Comment:
Shall we just add this person in the DESCRIPTION file and drop this line?  I think the input can be removed as well.

Just trying to standardize these files.

Reply:
Done. Think it fine to leave name out of DESCRIPTION file.